### ISSUE TITLE
[BUG] prefer-iterable-of warns on unrelated type from other package.

### ISSUE THREAD

**Issue by khoadng** [dart-code-checker/dart-code-metrics#1046]
* **Dart code metrics version:** 4.15.2
* **Dart sdk version:** 2.18.2

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```yaml
dart_code_metrics:
  rules:
    - prefer-iterable-of
```

</details>

**What did you do? Please include the source code example causing the issue.**
1. Install https://pub.dev/packages/collection
2. Use QueueList from the package

```dart
import 'package:collection/collection.dart';

void main() {
  final _ = QueueList.from([1, 2, 3]);
}

```

**What did you expect to happen?**
Analyzer shouldn't show warning.

**What actually happened?**
Analyzer show warning about replacing `from` to `of` on QueueList but that method doesn't exist on QueueList

**Are you willing to submit a pull request to fix this bug?**
Maybe

**Comment by lsaudon** [comment]
Which `collection` version do you use?

**Comment by incendial** [comment]
Available in 5.0.0 🚀
